src/main/java/appeng/parts/misc/ItemHandlerAdapter.java

```diff
@@ -91,7 +89,14 @@ public IAEItemStack injectItems( IAEItemStack iox, Actionable type, IActionSourc
 
 		if( type == Actionable.MODULATE )
 		{
-			this.onTick();
+			try
+			{
+				this.proxy.getTick().alertDevice( this.proxy.getNode() );
+			}
+			catch( GridAccessException ex )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)